test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Verb.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+﻿Describe "Get-Verb" -Tags "CI" {
+
+    It "Should get a list of Verbs" -test {
```

Comment:
We don't use `-test` anywhere else, so it just looks weird here and should probably be removed.